ns/context.go

```diff
@@ -0,0 +1,54 @@
+package ns
+
+import (
+	"context"
+	"net/http"
+	"strings"
+)
+
+// QriCtxKey defines a distinct type for
+// keys for context values should always use custom
```

Comment:
Missing a word here? I'm having a hard time reading this and understanding the intent.